### ISSUE TITLE
Corrupted recording cause readMetadata to go to endless loop (of sort)

### ISSUE THREAD

**Issue by solshark** [ClementBeal/audio_metadata_reader#33]
I have corrupted file which uploaded by user. Analyzing file with readMetadata leads readMetadata to not respond with anything and looks like it's going thru endless loop.

audio_metadata_reader: 0.0.10

```dart
debugPrint('fileSummary');
debugPrint(fileSummary.value.path);
debugPrint(meeting.toString());

final metadata = readMetadata(
  File(fileSummary.value.path),
  getImage: false,
);
debugPrint('MDATA FOUND');
debugPrint(metadata.duration.toString());
```
Line `debugPrint('MDATA FOUND');` never called.

Expected result: at least throw error.

File could be downloaded here: [dropbox](https://www.dropbox.com/scl/fi/xymm78lcouwx0w4x9xfb8/meeting-d7badd13-ebf1-4266-9782-8577042fd8db.m4a?rlkey=s7zoej7yd7yl0gvi3hf1u6cc1&st=z1yxwxpl&dl=0)

**Comment by ClementBeal** [comment]
Hi,

I found and fixed the bug. I'll deploying a new version later.

Thanks for reported the bug!

**Comment by ClementBeal** [comment]
Done with #35 

And available with the `1.0.0`
